src/openrpc/certificationextension.json

```diff
@@ -0,0 +1,383 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "CertificationExtension",
+        "description": "CertificationExtension provides various methods to get platform and/or app specific details which would be required for certification.",
+        "version": "0.0.0"
+    },
+    "methods": [
+        {
+            "name": "onFireboltInteractions",
+            "summary": "",
+            "tags": [
+                {
+                    "name": "event"
+                },
+                {
+                    "name": "capabilities",
+                    "x-provides": "xrn:firebolt:capability:certification:fireboltinteractions",
+                    "x-allow-focus": true
+                }
+            ],
+            "params": [],
+            "result": {
+                "name": "value",
+                "summary": "Result of the interaction",
+                "schema": {
+                    "$ref": "#/components/schemas/InteractionResult"
+                }
+            },
+            "examples": [
+                {
+                    "name": "Firebolt interaction to acquire platform token",
+                    "params": [],
+                    "result": {
+                        "name": "event",
+                        "value": {
+                            "interactionRequest": {
```

Comment:
probably could just have this be a json string instead individual fields being in the schema here.